server/routes/bookings/bookingDetailsController.ts

```diff
@@ -0,0 +1,48 @@
+import type { RequestHandler } from 'express'
+import { ValidationChain, param, validationResult } from 'express-validator'
+import { VisitService } from '../../services'
+import getPrisonInformation from '../../constants/prisonInformation'
+import paths from '../../constants/paths'
+
+export default class BookingDetailsController {
+  public constructor(private readonly visitService: VisitService) {}
+
+  public view(): RequestHandler {
+    return async (req, res) => {
+      const { booker, bookings } = req.session
+
+      const errors = validationResult(req)
+      if (!errors.isEmpty()) {
+        req.flash('errors', errors.array())
+        return res.redirect(paths.BOOKINGS.HOME)
+      }
+
+      const visitNumber = +req.params.visitNumber
+      const visit = bookings[visitNumber - 1]
+
+      // if manual number entered (larger than current number of visits in session data)
+      // redirect to bookings page
+      if (visitNumber > bookings.length) {
+        return res.redirect(paths.BOOKINGS.HOME)
+      }
+
+      const prisoner = booker.prisoners[0]
+
+      const { prisonName, prisonPhoneNumber, prisonWebsite } = getPrisonInformation(prisoner.prisonCode)
```

Comment:
Is this not available from any APIs anywhere? Just wondering if we could get this from a central place as we don't really want an app to be the source of truth for these details?

Reply:
it was agreed that for now we'd hard code these - thats why i've added them into a constant file (we were duplicating it across places)